src/webgpu/api/validation/createTexture.spec.ts

```diff
@@ -372,15 +373,20 @@ g.test('sampleCount,valid_sampleCount_with_other_parameter_varies')
       usage,
     };
 
+    const usableWithStorageUsage =
```

Comment:
Find this name a bit confusing.
Maybe something like `satisfyWithStorageUsageRequirement`, `storageUsageValidationFlag` etc

Reply:
done